wordpress-mobile/WordPress-iOS
# Remove `postScheduling` feature flag

#12686

To test:
- Check that Post Publish Date setting uses new Calendar UI for selecting a date.

PR submission checklist:

- [x] ~I have considered if this change warrants user-facing release notes and have added them to `RELEASE-NOTES.txt` if necessary.~ Already released in 14.0
